### PR TITLE
Bugfix/jon/1632 stop checking in pdfs

### PR DESCRIPTION
**Pull request by jericson-edb**
## What Changed?

* Stop checking in PDFs.
* Generate PDFs in the deploy-main and deploy-develop workflows instead.
* Update the README to warn folks they will need to remove and reclone local repositories.
* Remove all the PDFs we are generating so that we can verify they are being deployed correctly. (Still have the history in Git, but the files themselves are removed.)

### CODE REVIEW

File: .github/workflows/deploy-develop.yml [hunk 1]
```diff
@@ -3,14 +3,14 @@ on:
...
       - develop
+      - bugfix/jon/1632-stop-checking-in-pdfs
 jobs:
   build-deploy:
     runs-on: ubuntu-latest
     steps:
       - name: Cleanup disk
         uses: curoky/cleanup-disk-action@v2.0
-        with:
-          retain: "python,node"
```
Comment (jericson-edb) [inline .github/workflows/deploy-develop.yml:13]: We're going to reinstall these in a minute, but it doesn't hurt to remove the default versions.

File: README.md [hunk 2]
```diff
@@ -6,6 +6,12 @@
 
 This repo contains the React/Gatsby application that powers [the EDB Docs website](https://www.enterprisedb.com/docs/). The site pulls [Markdown](https://www.markdownguide.org/) content from several repos in a process called "sourcing", and then renders it all into high-performance markup. You can install the application on your local computer for easy editing, viewing, and eventually publishing to the GitHub repo.
 
+## Please remove and re-clone your local repositories after August 6, 2021
```
Comment (jericson-edb) [inline README.md:9]: Sooner we do this the better. I'm not sure waiting a week will do any good, so I suggest ripping the band aid off this weekend.

**Review by jericson-edb — COMMENTED** [review]
Pretty confident this will work, but we might as well find out for sure. ;-)

File: README.md (hunk 2 above)
Comment (gsong) [reply README.md:9]: That works. I would suggest that you create or sync a fork right before you blow the old repo away.

Also, we'll have to give notice to folks working on the docs repo that they'll need to be sure to push their changes up, or stash them away at the end of Friday.

**Comment by gsong** [comment]
💭 We should take this opportunity to upgrade to `actions/setup-node@v2`.

File: README.md [hunk 3]
```diff
@@ -6,6 +6,12 @@
...
+
+We've been checking PDF files into Git. That's not a good plan, so [we've stopped doing that](https://github.com/EnterpriseDB/docs/issues/1632). The next step is to remove these files from Git history and [move other large files to LFS](https://docs.github.com/en/github/managing-large-files/versioning-large-files/moving-a-file-in-your-repository-to-git-large-file-storage). Among many other good things, that ought to reduce the time to clone this repository substantially.
+
+But it comes at a cost. If there are any local repositories that were cloned before the change, we risk introducing dirty history back into the repository. So we're asking that everyone who has a local repository they **cloned before (or on) August 6, 2021** to delete those repositories. Unfortunately, we'll need to reject any pull requests that introduce PDF files back into Git history. (If you need any help with this, please contact jon.ericson@enterprisedb.com .)
```
Comment (gsong) [inline README.md:13]: 🎨 Remove the extra space between the email address and the period.

**Review by gsong — APPROVED** [review]
👏 Great work, @jericson-edb.

File: README.md (hunk 2 above)
Comment (jericson-edb) [reply README.md:9]: Creating a new fork makes sense. It's not as if we're trying to hide API keys or something. :-)

Tomorrow I'll send out something internally to people who have touched the repo. Not sure what to do about external contributors or even how many we might have.

**Review by josh-heyer — APPROVED** [review]
Looks simple and clean. 👍

File: .github/workflows/deploy-develop.yml [hunk 4]
```diff
@@ -28,6 +28,20 @@ jobs:
         env:
           NODE_ENV: ${{ secrets.NODE_ENV }}
 
+      - uses: actions/setup-python@v2
+        with:
+          python-version: "3.x"
+      - uses: r-lib/actions/setup-pandoc@v1
+        with:
+          pandoc-version: "2.10.1"
```
[the remark this answers is not in the document]
Comment (gsong) [reply .github/workflows/deploy-develop.yml]: @jericson-edb `develop` was just updated to 2.14.1: https://github.com/EnterpriseDB/docs/blob/develop/.github/workflows/update-pdfs-on-develop.yml#L31

😬

[the remark this answers is not in the document]
Comment (jericson-edb) [reply .github/workflows/deploy-develop.yml]: Got it. :-)

**Pull request merged**